src/utils/event-utils.ts

```diff
@@ -194,6 +202,7 @@ export const Info = {
                 $browser: Info.browser(userAgent, navigator.vendor),
                 $device: Info.device(userAgent),
                 $device_type: Info.deviceType(userAgent),
+                $timezone: Info.timezone(),
```

Comment:
@robbie-c would be interesting to add `$locale` as well
did not test but something like:
```
function getBrowserLocale(): string | undefined {
    try {
        return navigator.language || navigator.languages[0];
    } catch {
        return undefined;
    }
}
```